src/modules/bot/bot.service.ts

```diff
@@ -625,9 +627,23 @@ export class BotService {
     })
     .then()
     .catch((err) => {
-      this.logger.error(`Got failure response from inbound on cache invalidation endpoint ${caffine_reset_url}. Error: ${err}`);
+      this.logger.error(`Got failure response from inbound on cache invalidation endpoint ${caffine_reset_url_inbound}. Error: ${err}`);
       throw new ServiceUnavailableException('Could not invalidate cache after update!');
     });
+    await fetch(caffine_reset_url_orchestrator, {method: 'DELETE', headers: {'Authorization': transaction_layer_auth_token}})
```

Comment:
What is this doing in this PR?

Reply:
Orchestrator has a local cache of the bot data. This needs to be cleaned up when we change the notification data or else it will trigger the notification with old cached data. Earlier this function was only cleaning inbound service cache.